src/SDKs/SqlManagement/Management.Sql/Customizations/DatabaseVulnerabilityAssessmentRuleBaselinesOperations.cs

```diff
@@ -0,0 +1,688 @@
+// <auto-generated>
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for
+// license information.
+//
+// Code generated by Microsoft (R) AutoRest Code Generator.
+// Changes may cause incorrect behavior and will be lost if the code is
+// regenerated.
+// </auto-generated>
+
+namespace Microsoft.Azure.Management.Sql
+{
+    using Microsoft.Rest;
+    using Microsoft.Rest.Azure;
+    using Models;
+    using Newtonsoft.Json;
+    using System.Collections;
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Net;
+    using System.Net.Http;
+    using System.Threading;
+    using System.Threading.Tasks;
+
+    /// <summary>
+    /// DatabaseVulnerabilityAssessmentRuleBaselinesOperations operations.
+    /// </summary>
+    internal partial class DatabaseVulnerabilityAssessmentRuleBaselinesOperations : IServiceOperations<SqlManagementClient>, IDatabaseVulnerabilityAssessmentRuleBaselinesOperations
+    {
+        /// <summary>
+        /// Gets a database's vulnerability assessment rule baseline.
+        /// </summary>
+        /// <param name='resourceGroupName'>
+        /// The name of the resource group that contains the resource. You can obtain
+        /// this value from the Azure Resource Manager API or the portal.
+        /// </param>
+        /// <param name='serverName'>
+        /// The name of the server.
+        /// </param>
+        /// <param name='databaseName'>
+        /// The name of the database for which the vulnerability assessment rule
+        /// baseline is defined.
+        /// </param>
+        /// <param name='ruleId'>
+        /// The vulnerability assessment rule ID.
+        /// </param>
+        /// <param name='customHeaders'>
+        /// Headers that will be added to request.
+        /// </param>
+        /// <param name='cancellationToken'>
+        /// The cancellation token.
+        /// </param>
+        /// <exception cref="CloudException">
+        /// Thrown when the operation returned an invalid status code
+        /// </exception>
+        /// <exception cref="SerializationException">
+        /// Thrown when unable to deserialize the response
+        /// </exception>
+        /// <exception cref="ValidationException">
+        /// Thrown when a required parameter is null
+        /// </exception>
+        /// <exception cref="System.ArgumentNullException">
+        /// Thrown when a required parameter is null
+        /// </exception>
+        /// <return>
+        /// A response object containing the response body and response headers.
+        /// </return>
+        public async Task<AzureOperationResponse<DatabaseVulnerabilityAssessmentRuleBaseline>> GetWithHttpMessagesAsync(string resourceGroupName, string serverName, string databaseName, string ruleId, Dictionary<string, List<string>> customHeaders = null, CancellationToken cancellationToken = default(CancellationToken))
```

Comment:
Instead of duplicating code, can you just call `GetWithHttpMessagesAsync(resourceGroupName, serverName, databaseName, ruleId, VulnerabilityAssessmentPolicyBaselineName.Default)` ?

Reply:
Actually, you don't even need this. This is an `internal` class. You only need to implement the Extensions class with `Get` and `GetAsync`.

Reply:
Done.